packages/parse5-html-rewriting-stream/README.md

```diff
@@ -30,5 +30,5 @@
 </p>
 
 <p align="center">
-    <a href="https://github.com/inikulin/parse5/tree/master/docs/version-history.md">Version history</a>
+    <a href="https://github.com/inikulin/parse5/releases">Version history</a>
```

Comment:
Good to call all of these “Changelog” as well I think

Reply:
Good call